tests/foreman/cli/test_role.py

```diff
@@ -426,81 +431,832 @@ def test_positive_update_role_taxonomies(self):
+        """Users in usergroup can/cannot have access to the resources in
+        taxonomies depends on the taxonomies of Org Admin role is same/not_same
+
+        :id: 55099979-de11-4730-83ce-e190a3b8ecaa
+
+        :steps:
+
+            1. Create an Org Admin role by cloning 'Organization admin' role
+            2. Assign an organization A and Location A to the Org Admin role
+            3. Create an user without assigning roles while creating them and
+                assign Organization B and Location B
+            4. Create another user without assigning roles while creating them
+                and assign Organization A and Location A
+            5. Create an user group add above two users to the user group
+
+        :expectedresults:
+
+            1. User assigned to Organization B and Location B shouldn't have
+                access to the resources of organization A,B and Location A,B
+            2. User assigned to Organization A and Location A should have
+                access to the resources of organization A and Location A
+
+        :CaseLevel: System
+        """
+
+    @stubbed
+    @tier3
+    def test_positive_assign_org_admin_to_user_group(self):
+        """Users in usergroup can access to the resources in taxonomies if
+        the taxonomies of Org Admin role are same
+
+        :id: 07fa1bb4-1cce-4afa-a4f3-669704450947
+
+        :steps:
+
+            1. Create an Org Admin role by cloning 'Organization admin' role
+            2. Assign an organization A and Location A to the Org Admin role
+            3. Create two users without assigning roles while creating them
+            4. Assign Organization A and Location A to both users
+            5. Create an user group add above two users to the user group
+
+        :expectedresults: Both the user should have access to the resources of
+            organization A and Location A
+
+        :CaseLevel: System
+        """
+
+    @stubbed
+    @tier3
+    def test_negative_assign_org_admin_to_user_group(self):
+        """Users in usergroup can not have access to the resources in
+        taxonomies if the taxonomies of Org Admin role is not same
+
+        :id: 81c076ba-d61c-4d03-96be-6db8458a2470
+
+        :steps:
+
+            1. Create an Org Admin role by cloning 'Organization admin' role
+            2. Assign an organization A and Location A to the Org Admin role
+            3. Create two users without assigning roles while creating them
+            4. Assign Organization B and Location B to both users
+            5. Create an user group add above two users to the user group
+
+        :expectedresults: Both the user shouldn't have access to the resources
+            of organization A,B and Location A,B
+
+        :CaseLevel: System
+        """
+
+    @stubbed
+    @tier2
+    def test_negative_assign_taxonomies_by_org_admin(self):
+        """Org Admin doesn't have permissions to assign org/loc to any of
```

Comment:
Because context has been decided by Super Admin and not by org admin. 
Also the org admin is already scoped to some context and so the user created by org_admin will already have access to that context.